packages/admin/cms-admin/src/blocks/createRichTextBlock.tsx

```diff
@@ -1,4 +1,5 @@
 import { IRteOptions, makeRteApi, pasteAndFilterText, Rte } from "@comet/admin-rte";
+import { stateToHtml } from "@comet/admin-rte/lib/core/translation/stateToHtml";
```

Comment:
This should be part of the public API if it's used here.

Reply:
8c04fed0885bbd940f42a406c98fa149b7112d84